src/app/game/game-draw/game-draw.component.ts

```diff
@@ -273,7 +273,7 @@ export class GameDrawComponent implements OnInit, OnDestroy {
   private createDrawingTimer() {
     return new Observable((observer) => {
       let color = 'red';
-      const intervalDuration = 100;
+      const intervalDuration = this.gameConfigService.getConfig.intervalDuration;

```

Comment:
Maybe make a variable 'configValues' outside the method, and refer to that in here. This is good for scalability